coordinator/src/routes/util.ts

```diff
@@ -18,18 +19,36 @@ export function handlePollableResponse(spineResponse: SpineDirectResponse | Spin
   }
 }
 
-type Handler<T> = (requestPayload: T, responseToolkit: Hapi.ResponseToolkit) => Hapi.ResponseObject | Promise<Hapi.ResponseObject>
+type Handler<T> = (
+  requestPayload: T, responseToolkit: Hapi.ResponseToolkit
+) => Hapi.ResponseObject | Promise<Hapi.ResponseObject>
 
-export function validatingHandler(requireSignature: boolean, handler: Handler<Bundle>) {
+export type MessageType = "Prescription" | "Cancellation"
```

Comment:
this enum defines what type of message is sent in - potentially a better way of doing this?